internal/plugin/index_test.go

```diff
@@ -27,14 +29,18 @@ func TestValidateIndexErrors(t *testing.T) {
 			* entries[6]: 1 error occurred:
 				* field type cannot be empty
 			* entries[7]: 1 error occurred:
-				* field name cannot be empty`)
+				* field name cannot be empty
+			* entries[8]: 1 error occurred:
+				* field type is not valid, allowed values are [source executor]
+			* entries[9]: 1 error occurred:
+				* dependency URL for key "kubectl" in the URL entry "https://github.com/kubeshop/botkube/releases/download/v0.1.0/linux-_arm64_source_kubernetes" cannot be empty`)
```

Comment:
maybe we should use the `platform` and `arch` properties instead of URL? WDYT? 

as the URL in some edge cases can be the same (https://en.wikipedia.org/wiki/Fat_binary)